From: chrysn <chrysn@fsfe.org>
Date: Mon, 8 Jun 2015 09:58:38 +0200
Subject: pack OPENSCAD_NOGUI in qmake nogui config

thus, scintilla can be kept from being built at all, slimming down the
build dependencies for the nogui build
---
 openscad.pro         | 8 +++++++-
 scripts/travis-ci.sh | 2 +-
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/openscad.pro b/openscad.pro
index 5234dbb..bec4f0e 100644
--- a/openscad.pro
+++ b/openscad.pro
@@ -207,13 +207,19 @@ CONFIG += fontconfig
 CONFIG += gettext
 
 #Uncomment the following line to enable the QScintilla editor
-CONFIG += scintilla
+!nogui {
+  CONFIG += scintilla
+}
 
 # Make experimental features available
 experimental {
   DEFINES += ENABLE_EXPERIMENTAL
 }
 
+nogui {
+  DEFINES += OPENSCAD_NOGUI
+}
+
 mdi {
   DEFINES += ENABLE_MDI
 }
diff --git a/scripts/travis-ci.sh b/scripts/travis-ci.sh
index a808a12..e4160f9 100755
--- a/scripts/travis-ci.sh
+++ b/scripts/travis-ci.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-qmake CONFIG+=experimental DEFINES+=OPENSCAD_NOGUI
+qmake CONFIG+=experimental CONFIG+=nogui
 make
 
 cd tests
